### PR TITLE
Remove climate content from landing page template

### PR DESCRIPTION
**Pull request by wslack**
cc @jeremiak to make sure I didn't miss anything.

In general, this will need more work to be "federalist ready" given how confusing it is to look at in the web editor right now.

See the sample site here: https://federalist.18f.gov/preview/18F/federalist-landing-page-template/remove-climate/

### CODE REVIEW

File: _includes/report.md [hunk 1]
```diff
@@ -1,45 +1,22 @@
...
-  Currently, climate-change planners lack a single connecting
-  website that helps them find locally relevant federal tools. For
-  greatest success, such a site should provide locally relevant
-  data, and should focus on curating resources rather than
-  cataloguing everything that’s available.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sodales libero sit amet sapien accumsan bibendum. Quisque vestibulum ultricies sapien sed porta. Proin ipsum tellus, facilisis eget luctus sed, sollicitudin in massa. Mauris eu semper neque. Cras tempor suscipit odio sit amet maximus. Nunc ante lectus, dictum non fringilla sed, aliquet at odio. Etiam ut lorem eu est mollis semper. Vivamus porttitor luctus lacinia.
 
+## Sections of information
+
+* ### Planning
```
Comment (wslack) [inline _includes/report.md:7]: @jeremiak this is the change that must have removed the logo icons...but how?

**Comment by el-mapache** [comment]
@wslack I updated the package.json to remove references to the climate site there as well. I also bumped the `uswds` dependency to the latest version. I also fixed the icons and layout.

Just noticed brandon was assigned, sorry about that. I thought you asked me to take a look at this last week! Sorry if i stepped on any toez 👣

**Comment by wslack** [comment]
all good!

**Comment by wslack** [comment]
Federalist says: 

> Your Ruby version is 2.3.1, but your Gemfile specified 2.2.3

Is something else needed here?

**Comment by jmhooper** [comment]
@el-mapache: You'll need to run `bundle install` as well to update the `Gemfile.lock`

**Comment by el-mapache** [comment]
ah yes, forgot about that

**Pull request merged**